p/themes/Ansum/ansum.rtl.css

```diff
@@ -1137,9 +1133,6 @@ main.prompt {
 .post form {
 	margin: 1rem 0;
 }
-.post form .horizontal-list {
-	margin-bottom: 0.5rem;
-}
```

Comment:
I would be somewhat careful with a few details like this one as a rule of thumb. It's not inconceivable that an extension expects to be able to use it.

But in this case it shouldn't be very harmful. ^_^